Jenkinsfile

```diff
@@ -40,6 +61,8 @@ def namespace    = 'catalog'
 def root_path    = 'src/github.com/kubernetes-incubator/service-catalog'
 
 node {
+  echo 'Testingd'
```

Comment:
This PR was purely experimental. That whole line was me iterating on commits to trigger new builds on my fork, and the additional letter was an easy way to distinguish between them, haha.